src/mscorlib/src/System/Collections/Hashtable.cs

```diff
@@ -66,6 +66,7 @@ namespace System.Collections
     // 
     [DebuggerTypeProxy(typeof(System.Collections.Hashtable.HashtableDebugView))]
     [DebuggerDisplay("Count = {Count}")]
+    [Serializable]
```

Comment:
This is not going to work because of it will conflict with public Hashtable in corefx. When deserializing payload from desktop, the binary serializer will not know whether it needs to create this internal Hashtable or public Hashtable in corefx.

What is this needed for? If it is really needed, I think it needs to be fixed by moving the Hashtable from corefx to corelib.